src/main/java/uk/gov/pay/adminusers/service/LinksBuilder.java

```diff
@@ -29,8 +29,7 @@ public User decorate(User user) {
     }
 
     public Service decorate(Service service) {
-        //TODO: fix to use the externalId, after getting rid of the primary key based GET
```

Comment:
This comment made me search around for places using `findById` and came across a few examples like https://github.com/alphagov/pay-adminusers/blob/b95501107acdbf22625f7da73e860a4067befdbb/src/main/java/uk/gov/pay/adminusers/resources/ServiceResource.java#L153

It seems we are still accounting for the possibility of having a numeric (internal) id - it would be good to clean those places up